package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ngx-maplibre-gl-srcs",
-  "version": "17.4.0",
+  "version": "17.4.1",
```

Comment:
There is an action that will take care of it, I'll run it right after I merge this.

Reply:
```suggestion
  "version": "17.4.0",
```